### ISSUE TITLE
Keyboard focus is not visible on column header of expense report table after applying high contrast:A11y_WPF Runtime_ExpenseItIntro_User can get all insights from report of any name_HighContrast

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/wpf#10945]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WPF; #BM_WPFRuntime_Win32_JUNE2025; #WCAG2.4.7; #Win32; #DesktopApp; #FTP; #Benchmark; #HighContrast; #Closed;

## Environment Details:
Application Name:  WPF 
Version
OS: Windows 11[Version 23H2(OS Build 22631.3593)]  
### Prerequisites to apply High Contrast

1. Open settings, tab till accessibility and press enter.
2. Tab till contrast theme list item and press enter.
3. Tab till contrast themes dropdown and press enter then select 'Aquatic' theme.
4. Tab till 'apply' button and press enter.

## Repro Steps:

1. Open ExpenseItIntro folder from the extracted [WPF-NET10-A11y-Testing.zip](https://microsoftapc-my.sharepoint.com/:u:/g/personal/dipeshkumar_microsoft_com/Eafz-xf_fLpMnlal9jN60FgBUz9Jll5eQbafO494M4yEcA?e=BwsKbH&xsdata=MDV8MDJ8di1rdmFpc2huYXZAbWljcm9zb2Z0LmNvbXwwMWM0NzA3ZmRhOTU0YmIxMDdmNzA4ZGRhZDk0NTllZXw3MmY5ODhiZjg2ZjE0MWFmOTFhYjJkN2NkMDExZGI0N3wxfDB8NjM4ODU3NTc0ODc3Njc4MDg5fFVua25vd258VFdGcGJHWnNiM2Q4ZXlKRmJYQjBlVTFoY0draU9uUnlkV1VzSWxZaU9pSXdMakF1TURBd01DSXNJbEFpT2lKWGFXNHpNaUlzSWtGT0lqb2lUV0ZwYkNJc0lsZFVJam95ZlE9PXwwfHx8&sdata=ckdjUTR1T1R3WVZlWjRhUG5talJuKzRpS2RCbnk1R1hQaFA4bitKSnovYz0%3d) Zip File
2. TAB till ExpenseItIntro exe file and open it.
3. "Expenselt" Window gets opened.
4. TAB till name list
5. TAB till view button and press ENETR key.
6. Apply High contrast
7. TAB till Column header of table
8. Observe the issue 

## Actual Result: 
Keyboard focus is not visible on column header of expense report table after applying high contrast.

### Similar issue repro in following scenario:

1. Launch Edit Examiner Demo folder form [WPF-NET10-A11y-Testing.zip](https://microsoftapc-my.sharepoint.com/:u:/g/personal/dipeshkumar_microsoft_com/Eafz-xf_fLpMnlal9jN60FgBUz9Jll5eQbafO494M4yEcA?e=BwsKbH&xsdata=MDV8MDJ8di1rdmFpc2huYXZAbWljcm9zb2Z0LmNvbXwwMWM0NzA3ZmRhOTU0YmIxMDdmNzA4ZGRhZDk0NTllZXw3MmY5ODhiZjg2ZjE0MWFmOTFhYjJkN2NkMDExZGI0N3wxfDB8NjM4ODU3NTc0ODc3Njc4MDg5fFVua25vd258VFdGcGJHWnNiM2Q4ZXlKRmJYQjBlVTFoY0draU9uUnlkV1VzSWxZaU9pSXdMakF1TURBd01DSXNJbEFpT2lKWGFXNHpNaUlzSWtGT0lqb2lUV0ZwYkNJc0lsZFVJam95ZlE9PXwwfHx8&sdata=ckdjUTR1T1R3WVZlWjRhUG5talJuKzRpS2RCbnk1R1hQaFA4bitKSnovYz0%3d) Zip File
2. TAB till Edit Examiner Demo exe file and open it.
3. TAB till Now exception text box used to show error
4. Observe that keyboard focus is not visible

## Expected Result:
When navigating with the TAB key, the keyboard focus should not move to the entire column header section as it is non-actionable. Additionally, when the focus moves to the first and second column header, the keyboard should be visible in high contrast mode.

## User Impact:
Low vision user who uses application in high contrast mode will not able able to see keyboard focus when moves through column header section on pressing tab key.

## Attachment
Aquatic Mode

https://github.com/user-attachments/assets/ac0a170a-818b-401e-bc4e-d728720fe11e

Desert mode

https://github.com/user-attachments/assets/0f913599-df03-49bb-83db-c5e214b5a6ca

### Similar issue attachment
Aquatic Mode

https://github.com/user-attachments/assets/40e8586e-213f-49b4-b41d-e31387ab2b20

Desert mode

https://github.com/user-attachments/assets/a0b5b90a-1a00-4466-9d37-6f9b7bc06741

**Comment by kapilvaishna** [comment]
Closing as logged new bug: https://github.com/microsoft/WPF-Samples/issues/709
